addons/mail/static/tests/owl/component/chatter_topbar_tests.js

```diff
@@ -50,19 +50,94 @@ QUnit.module('ChatterTopbar', {
+                return new Promise(() => {});
+            }
+            return this._super(...arguments);
+        }
+    });
+    const chatterLocalId = this.env.store.dispatch('createChatter', {
+        initialThreadId: 100,
+        initialThreadModel: 'res.partner',
+    });
+    await this.createChatterTopbar(chatterLocalId, { isDisabled: false });
+
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar`).length,
+        1,
+        "should have a chatter topbar"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonSendMessage`).length,
+        1,
+        "should have a send message button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonLogNote`).length,
+        1,
+        "should have a log note button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonScheduleActivity`).length,
+        1,
+        "should have a schedule activity button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonAttachments`).length,
+        1,
+        "should have an attachments button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonAttachmentsCountLoader`).length,
+        1,
+        "attachments button should have a loader"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonAttachmentsCount`).length,
+        0,
+        "attachments button should not have a counter"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonFollow`).length,
+        1,
+        "should have a follow button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonFollowers`).length,
+        1,
+        "should have a followers button in chatter menu"
+    );
+    assert.strictEqual(
+        document.querySelectorAll(`.o_ChatterTopbar_buttonFollowersCount`).length,
+        1,
+        "followers button should have a counter"
+    );
+});
+
+QUnit.test('base rendering (directly loaded attachments)', async function (assert) {
+    assert.expect(10);
 
+    const attachmentPromise = makeTestPromise();
     await this.start({
         async mockRPC(route) {
             if (route.includes('ir.attachment/search_read')) {
+                attachmentPromise.resolve();
                 return [];
             }
             return this._super(...arguments);
         }
     });
-    const threadLocalId = await this.createThread({ _model: 'res.partner', id: 100 });
-    await this.createChatterTopbar(threadLocalId);
+    const chatterLocalId = this.env.store.dispatch('createChatter', {
+        initialThreadId: 100,
+        initialThreadModel: 'res.partner',
+    });
+    await this.createChatterTopbar(chatterLocalId, { isDisabled: false });
```

Comment:
don't understand why you pass `isDisabled`, should crash because this props is not defined in static `props`

Reply:
Reminiscence of when isDisabled was a props, now it's a storeProps